pluginManager/src/PluginList.cpp

```diff
@@ -383,10 +395,14 @@ void PluginList::addSteps(Plugin* plugin, TiXmlElement* installElement, InstallO
 	{
 		// If it is a unicode tag, then only process the contents if it's a unicode N++
 		// or if it's an ansi tag, then only process the contents if it's an ansi N++
-		if ((g_isUnicode 
-			&& !_tcscmp(installStepElement->Value(), _T("unicode")) 
+		if ((g_isUnicode && g_isX64

```

Comment:
Sounds good. Let's run with `<x64>` and `<x64Version>`, leave unicode as is for now.

I'm working on the new admin code.